themes/mlibrary_new/exhibit-builder/exhibits/gallery.php

```diff
@@ -0,0 +1,45 @@
+
+<section class="row">
+  <div class="col-xs-12 col-sm-9">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item"><?php echo link_to_exhibit(metadata('exhibit','title',array('no_escape' => true)));?></li>
+        <li class="breadcrumb-item active"><?php echo 'Exhibit Image'. metadata('exhibit_page','title',array('no_escape' => true)); ?></li>
+      </ol>
+  </div>
+  <div class="col-xs-12 col-sm-3">
+    <ul class="share-this">
+      <li>Share this Gallery:</li>
+      <li><a href="https://twitter.com/share" class="twitter-share-button" data-text="I just saw '<?php echo metadata('exhibit','title',array('no_escape' => true)); ?>' at the U-M Library Online Exhibits!" ><span class="sr-only">Tweet</span> </a></li>
+    </ul>
+   </div>
+</section>
+
+<div>
+<?php echo "Exhibit Image Gallery";
```

Comment:
Lines 18-21 are echoing string literals, you shouldn't break into php to do that.

Reply:
This should be styled, I am just adding the content here.